### DISCUSSION TITLE
Smoke falls back down after exiting through the smoke vent - system with mechanical air supply

### DISCUSSION THREAD

**Discussion by ziuzu** [firemodels/fds#13458] [Q&A] [answered]
Hi, I have a problem evaluating the results of a smoke ventilation simulation for a staircase with mechanical air supply. The staircase has 18 stories. The simulation was conducted under summer conditions:
•	Indoor temperature: 24°C
•	Outdoor temperature: 28°C.
The issue is that after the smoke exits through the smoke vent, it doesn't rise further and flow out through the upper “OPEN VENT” of the model but instead falls back down. Screenshots are below:
![HIGH](https://github.com/user-attachments/assets/e02d9c6b-a9b1-402a-807f-cdf2926c04a4)

In previous simulations of staircase smoke ventilation systems, the smoke continued to rise after passing through the smoke vent and didn't settle on the roof. Below are screenshots from another example simulation, with a lower building—8 stories (also under summer temperature conditions):
![LOWER](https://github.com/user-attachments/assets/36c3abee-5f99-4013-b721-039e1ff170b3)

Does this make sense from a physics perspective? The smoke is no longer hot; it has a temperature close to the ambient temperature, so after passing through the vent, it has no reason to rise and instead falls back down? I obtained similar results for a higher fan output—7,200 m³/h—the smoke also fell back down after exiting through the vent.
For the discussed staircase, the results under winter and isothermal conditions are standard, meaning the smoke rises after exiting through the vent.
What causes this type of smoke behavior through the vent? Why do the results for the discussed staircase (18 stories) differ from those for the 8-story staircase? Is the answer simply the height of the buildings? Do these results seem plausible?
Attached is the FDS file.

[FILE.zip](https://github.com/user-attachments/files/17094596/FILE.zip)

**Comment by drjfloyd** [comment, marked as answer]
The exterior pressure is going to drop hydrostatically, and 18 stories will have a larger drop than 8 stories. This might be the cause. I suggest you add a slice file for density and look to see what is the density of the smoke leaving the building vs. the air above the top.

**Comment by Er9y714** [comment]
As @drjfloyd mentioned, density might be a good check. The product mix gas of the combustion might be denser than the ambient air depending on the stoichiometry (e.g. CO2 is heavier than the air).
